src/models/Token/integration.spec.js

```diff
@@ -0,0 +1,59 @@
+import resetDatabase from '@tests/resetDatabase'
+import database from '@database/index'
+import { createToken, retrieveToken, validateToken } from './index'
+
+describe('The Token model helpers', () => {
+  beforeAll(async done => {
+    await resetDatabase()
+    await database.migrate.latest()
+    done()
+  })
+
+  it("should return errors if the user doesn't exist", async done => {
+    const { token, errors } = await createToken()
+    const expected = { field: 'user', message: 'User should be defined' }
+
+    expect(token).toEqual(undefined)
+    expect(errors).toContainEqual(expected)
+    done()
+  })
+
+  it('should add a token to the database and return it', async done => {
+    await database.seed.run()
```

Comment:
If you run seeds in those 3 tests, you should move it to a `beforeEach` hook instead.
You probably need to reset the database between every test, too?